OrbitCore/TracepointEventBuffer.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CORE_TRACEPOINT_EVENT_BUFFER_H_
+#define ORBIT_CORE_TRACEPOINT_EVENT_BUFFER_H_
+
+#include <set>
+
+#include "BlockChain.h"
+#include "SamplingProfiler.h"
+#include "Threading.h"
+#include "capture_data.pb.h"
+
+class TracepointEventBuffer {
+ public:
+  TracepointEventBuffer() : m_MaxTime(0), m_MinTime(LLONG_MAX) {}
```

Comment:
`LLONG_MAX` -> `std::numeric_limits<uint64_t>::max()` to be sure the type fits. One can't be sure "long long"  is the same as "uint64_t"

Add `#include <limits>` on the top.